src/views/Statistic/StatisticGroup.js

```diff
@@ -0,0 +1,68 @@
+import _ from 'lodash'
+import cx from 'classnames'
+import React, { PropTypes } from 'react'
+
+import { customPropTypes, getUnhandledProps, useKeyOnly } from '../../utils/propUtils'
+import numberToWord from '../../utils/numberToWord'
+import META from '../../utils/Meta'
+import Statistic from './Statistic'
+
+function StatisticGroup(props) {
+  const { children, className, horizontal, items, widths } = props
+  const classes = cx(
+    'ui',
+    useKeyOnly(horizontal, 'horizontal'),
+    _.isNumber(widths) ? numberToWord(widths) : widths,
```

Comment:
Currently, if `widths='5'` is provided, the class `5` will be added because it isn't of type number.  I think we should support widths as strings as well.  This will be common when getting the number from an API or from the value of a DOM control.

One easy way to do this is to change the return statement in `numberToWord` to return the argument itself if it did not have a map for the number:

```diff
-return map[number]
+return map[number] || number
```

This way, `number` can be a string or number and it will still find the word equivalent.  If there is no match found (i.e. in case the user passed `two`) then it will be returned.